_targets.R

```diff
@@ -24,13 +27,24 @@ list(
   # static branching with tar_map()
   tar_map(
     ## task names passed into tar_map() via arg. 'values='
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+
     tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
                                              state_cd == state_abb)),
 
     tar_target(nwis_data, get_site_data(nwis_inventory,
                                         state_abb,
-                                        parameter))
+                                        parameter)),
+
+    tar_target(tally, tally_site_obs(nwis_data)),
+
+    tar_target(timeseries_png, plot_site_data(out_file = state_plot_files,
+                                              site_data = nwis_data,
+                                              parameter = parameter)
+    ),
+
+    names = state_abb
```

Comment:
Although the issue says to add `names` as the final argument to `tar_map()`, in my opinion it is better from a readability standpoint to have it up above right after `values` is defined, so that someone encountering the pipeline for the first time can easily see how the target branches will be named.